www/js/types/appConfigTypes.ts

```diff
@@ -0,0 +1,22 @@
+// WIP: type definitions for the 'dynamic config' spec
+// examples of configs: https://github.com/e-mission/nrel-openpath-deploy-configs/tree/main/configs
+
+export type AppConfig = {
+  server: ServerConnConfig;
+  [k: string]: any; // TODO fill in all the other fields
+};
```

Comment:
Might as well add survey_info to the AppConfig type since you already typed the survey config
```suggestion
export type AppConfig = {
  server: ServerConnConfig;
  survey_info: {
    'trip-labels': 'MULTILABEL' | 'ENKETO';
    surveys: EnketoSurveyConfig;
  }
  [k: string]: any; // TODO fill in all the other fields
};
```